Makefile

```diff
@@ -10,17 +10,48 @@ GO_LD_EXTRAFLAGS :=-X github.com/uor-framework/client/cli.version="$(shell git t
 				   -X github.com/uor-framework/client/cli.commit="$(GIT_COMMIT)" \
 				   -X github.com/uor-framework/client/cli.buildDate="$(shell date -u +'%Y-%m-%dT%H:%M:%SZ')"
 
-build:
-	mkdir -p ${GO_BUILD_BINDIR}
+build: prep-build-dir
```

Comment:
The default build should be on the platform that is performing the build. Should be able to remove `GOOS=linux GOARCH=amd64`